service/storage_proxy.cc

```diff
@@ -4902,8 +4902,7 @@ future<> storage_proxy::truncate_blocking(sstring keyspace, sstring cfname) {
 
 void storage_proxy::init_messaging_service(shared_ptr<migration_manager> mm) {
     _mm = std::move(mm);
-    auto& ms = _messaging;
```

Comment:
It was a way to make lines shorter. Not really related to eliminating captures.

Reply:
Yes, but ms was captured below. The patch converted it to capture `this` so it can later be a member function.